### DISCUSSION TITLE
Git-lesson Discussion

### DISCUSSION THREAD

**Discussion by Nightowl5275** [Chargers-4189/git-lesson#6] [General]
Here Emily, Aiden, and Jack coordinate for the stock code

**Comment by Nightowl5275** [comment]
Hello, respond to this is you can see this.

**Reply by Nightowl5275** [reply]
@almort08 @jlkoster08  Respond PLEASE

**Reply by almort08** [reply]
# responded #

**Reply by jlkoster08** [reply]
aahhhhhhhhhhhhhhhhhhh

**Comment by Nightowl5275** [comment]
I am editing the code I have to change the average and clear out old data. After that, we can edit the company names to the new ones that caleb added. Will y'all look at my code after I make it? I will probably be done by the coding meeting today.

**Comment by Aviator2276** [comment]
Hey Emily,

Thanks for starting the discussion. You can also ping members or tag them in comments so they're notified: @Aviator2276.

**Comment by Nightowl5275** [comment]
@almort08 @jlkoster08 @Aviator2276 I am the branch maintainer now. Does anybody have objections? I need to learn how to make branches.

**Reply by Nightowl5275** [reply]
I created a new branch called 24-app/feature/updated-to-new-rules. Could you both look at the code and see if there is anything you think we should change? @almort08 @jlkoster08

**Comment by almort08** [comment]
what are the roles

**Reply by almort08** [reply]
I am the branch maintainer

**Reply by Nightowl5275** [reply]
I am finding the averages and making variables for the volatility for each company.

**Reply by Nightowl5275** [reply]
we are all developers

**Reply by jlkoster08** [reply]
coder, debugger, and reviewer

**Reply by Nightowl5275** [reply]
whoever did not write the code is one of the two pull request reviewers

**Reply by jlkoster08** [reply]
I can do the code that will buy and sell I'll just talk with you guys for the logic and the amounts

**Comment by Nightowl5275** [comment]
I committed the averaging code.

**Reply by almort08** [reply]
Alright, thanks!
